avm/res/network/private-endpoint/main.bicep

```diff
@@ -15,31 +15,33 @@ param applicationSecurityGroupResourceIds array?
 param customNetworkInterfaceName string?
 
 @description('Optional. A list of IP configurations of the private endpoint. This will be used to map to the First Party Service endpoints.')
-param ipConfigurations ipConfigurationsType
+param ipConfigurations ipConfigurationsType[]?
```

Comment:
Did we eventually discuss on the option of making type names singular? This comment applies to all non common types below

Reply:
We talked about, but did not release a spec for it yet. I can update the wording for sure. In that moment my main focus was on definitely moving the array & nullable indicators 😄

Reply:
Updated @eriqua